.azure-pipelines.yml

```diff
@@ -13,6 +13,8 @@ pr:
   paths:
     exclude:
     - Docs
+    - .github/workflows/**
+    - .azure-pipelines.yml
```

Comment:
These two paths patterns are here for testing purposes only (since this PR modifies also files that match those patterns). They should be removed before merging the PR.